README.md

```diff
@@ -268,15 +268,15 @@ Example of conversions from the Java type ways:
 
 ```scala
 import scala.concurrent.duration._
-import scala.compat.java8.DurationConverters
+import scala.compat.java8.DurationConverters._
 
 val javaDuration: java.time.Duration = 5.seconds.toJava
 val finiteDuration: FiniteDuration = javaDuration.toScala
 ```
 
 From Java:
 ```java
-import scala.compat.java8.DurationConverters;
+import scala.compat.java8.DurationConverters._;
```

Comment:
isn't the Java syntax something else? like `*` instead of `_`? and should it be `import static` perhaps?

Reply:
Ooops yes, it should be:

```
import static scala.compat.java8.DurationConverters.*;
```

Reply:
I have fixed and rebased.

Reply:
Actually, the java import was correct as is, I have reverted back, sorry.